lib/token.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+module.exports = Token
+
+const Client = require('./client')
+const moment = require('moment')
+const orm = require('./connection')
+const uuid = require('uuid')
+
+function Token (opts) {
+  this.id = opts.id
+  this.client_id = opts.client_id
```

Comment:
`opts` may have a `client` property prepopulated with a `Client` instance if a join produced this instance — I've taken to [storing them as `_`-prefixed properties](https://github.com/npm/user-acl-two/blob/master/lib/models/user-token.js#L17) and [adding a `getter/setter` pair](https://github.com/npm/user-acl-two/blob/master/lib/models/user-token.js#L42-L51) that returns a promise for a `Client` instance (this is also handy, from bluebird you can do `Token.objects.get(...).get('client')` and get a promise for the associated client object).